source/daq/frequency_mask_trigger.cc

```diff
@@ -75,7 +80,19 @@ namespace psyllid
         LDEBUG( plog, "Setting threshold (power via power) to " << f_threshold_snr_high );
         return;
     }
+    void frequency_mask_trigger::set_threshold_power_sigma( double a_power_sigma )
```

Comment:
@nsoblath also here, we have several set methods that could just as well be `mv_asccessible` rather than `mv_accessible_noset`. Do you prefer to make that change for the setters that just implement `f_<thing> = a_<thing>`, or do you like keeping the ways of setting threshold all parallel, since some of them do require a calculation, and the function names here differ from the variables, again to be parallel with other sets.

Reply:
If the sets are just normal sets, with no other calculations made or variables affected, then `mv_accessible` should be used.  Custom set functions should only be used where they're needed.

Reply:
this should be fixed